tests/base/test_extractors.py

```diff
@@ -189,6 +189,33 @@ def test_duckling_entity_extractor(component_builder):
     assert entities[0]["text"] == "tomorrow"
     assert entities[0]["value"] == "2013-10-13T00:00:00.000Z"
 
+    # Test dimension filtering for numbers, as times are already tested above
+    _config = RasaNLUModelConfig(
+        {"pipeline": [{"name": "DucklingHTTPExtractor"}]}
+    )
+    _config.set_component_attr(0, dimensions=["number"], timezone="UTC",
```

Comment:
```suggestion
    _config.set_component_attr(0, dimensions=["number"],
```
Probably wouldn't specify a time zone if you're not parsing times 😄